manager.py

```diff
@@ -271,6 +296,14 @@ def get_interval(self, start_time):
         end_time = start_time + self.interval
         return start_time, end_time
 
+    def data_log(self, file, cpm, cpm_err):
+        """Writes cpm to data-log"""
+        if self.data:    
+            f = open(file, 'a')
+            f.write('{0}, {1}, {2}'.format(time.strftime("%Y-%m-%d %H:%M:%S"), cpm, cpm_err))
+            f.write('\n')
+            f.close()
+    
```

Comment:
Use `with` here. And maybe assign `time.strftime(...)` into a variable for readability so you can do `format(time_string, cpm, cpm_err)`